src/main/java/com/bbaker/discord/redmarket/commands/roll/RedMarketCommand.java

```diff
@@ -175,6 +179,31 @@ private int getInt(String val) throws BadFormatException {
         }
     }
 
+    private String[] extractDiceFromMessage(String diceMessage) {
+        Matcher matcher = diceFaceRgx.matcher(diceMessage);
+        String colour = null;
+        String face = null;
+
+        if(matcher.find()){
+            switch(matcher.group(1)) {
+                case "r":
+                case "red":
+                case "b":
+                case "black":
+                    colour = matcher.group(1);
+                    face = matcher.group(2);
+                    break;
+                default:
+                    colour = matcher.group(2);
+                    face = matcher.group(1);
+            }
+
+            System.out.println(colour + "--> " + face);
```

Comment:
So I'm trying to remove `System.out.println` from the code. I'm aware that my code still has two pre-existing `System.out.println` and I will be removing them soon. 

Please do the same

Reply:
No problem :+1: